doc/api/http2.md

```diff
@@ -874,6 +870,16 @@ stream.on('trailers', (headers, flags) => {
 });
 ```
 
+#### Event: 'trailers-ready'
```

Comment:
Considering that the other side of the duplex can have trailers as well, maybe we need a better name for this, or maybe just emit `'finish'` in the cases it's a `respondWithFile()`

Reply:
We *could* but when using `respondWithFile()`, one of the things that happens automatically is that `end()` is called on the stream API interface to ensure that someone does not attempt to write to the `Duplex` after calling `respondWithFile()`... I believe that will have the result of causing the stream to emit `'finish'` as part of the stream API processing. I'd prefer to have a single event that would work for both cases so I'm open to names.

Reply:
We could use the `'finish'` event if we change the way we handle closing the `Duplex` when `respondWith*` is used. Basically, don't end the Writable side but error if `write()` is used after `respondWith*` is used.

Reply:
+1 for two separate things.

I'm not necessarily certain that calling `end()` is the right thing to do in that case. But that's another issue.